next-tavla/app/(admin)/edit/[id]/components/DuplicateBoard/index.tsx

```diff
@@ -14,23 +15,27 @@ function DuplicateBoard({
     oid?: TOrganizationID
     type?: 'button' | 'action'
 }) {
+    const { addToast } = useToast()
     const handleSelect = async () => {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
trenger vi denne eller funker det uten?

Reply:
Usikker på om det er noe annen måte å fjerne IDen fra objektet på. Den eslit-disable er der for at den ikke skal klage på at id ikke blir brukt videre.

Reply:
Kanskje ikke uten å lage et nytt objekt, typ: 

```
const removeId = ({ id, ...rest }) => rest;
const boardWithoutId = removeId(board);
```
tenker det ikke har mye å si hva vi gjør 🤷

Reply:
Man kan bruke `delete` operatoren

```ts
delete board.id
await create({...)
```